src/components/Modals/NewCourse/NewCourseModal.vue

```diff
@@ -92,9 +61,6 @@ export default Vue.extend({
       year: 0,
```

Comment:
they can also be removed now, and then Semester.vue no longer needs to handle addCourse emit based on whether season and year having dummy value!